src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,9 +8,20 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+	private Block block;
+	private Street street;
+	private Unit unit;
+	private PostalCode postalCode;
+	
+	private static final int BLOCK_INDEX = 0;
+	private static final int STREET_INDEX = 1;
+	private static final int UNIT_INDEX = 2;
+	private static final int POSTALCODE_INDEX = 3;
+	
+    public static final String EXAMPLE = "123, Clementi Ave 3, #12-34, 231534";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can only be entered in the format: "
+    		+ "a/BLOCK, STREET, UNIT, POSTAL_CODE";
+    public static final String ADDRESS_VALIDATION_REGEX = ".+,.+,.+,.+";
```

Comment:
Watch the indentation please?  It's the first thing that an external reader notices...